api/clients/v2/verification/eigenda_cert.go

```diff
@@ -12,3 +17,30 @@ type EigenDACert struct {
 	BatchHeader                 contractEigenDABlobVerifier.BatchHeaderV2
 	NonSignerStakesAndSignature contractEigenDABlobVerifier.NonSignerStakesAndSignature
 }
+
+// BuildEigenDACert creates a new EigenDACert from a BlobInclusionInfo, BatchHeader, and NonSignerStakesAndSignature
+//
+// For convenience, this function accepts arguments as protobufs where applicable, since that's the form the caller
+// will have.
+func BuildEigenDACert(
```

Comment:
Feel like we should also have a `BuildEigenDACertFromBlobStatusReply` or something like that

Reply:
I don't think there is any need to have both, construction is really only going to happen in one place in production. I changed the existing function signature to accept a `BlobStatusReply` instead of the component parts, since I don't have a preference here [32862cf3](https://github.com/Layr-Labs/eigenda/pull/1159/commits/32862cf30144ebc70c582a22385c95408b3d1785)

I'm assuming you didn't mean the create a function `BuildEigenDACertFromBlobStatusReply` that does the fetching of the `NonSignerStakesAndSignature` from the eth client? IMO that would not be a good change.

Reply:
Don't remember 100% but think that IS what I meant lol. But didn't see that we already have the payloadDisperser.buildEigenDACert function that does this.